underfs/ozone/src/main/java/alluxio/underfs/ozone/OzoneUnderFileSystemFactory.java

```diff
@@ -30,7 +32,15 @@ public class OzoneUnderFileSystemFactory extends HdfsUnderFileSystemFactory {
 
   @Override
   public boolean supportsPath(String path) {
-    return path != null && path.startsWith(Constants.HEADER_OZONE);
+    AlluxioConfiguration conf = new InstancedConfiguration(ConfigurationUtils.defaults());
```

Comment:
It's expensive to me if you only want `UNDERFS_OZONE_PREFIXES`, why don't you create constants with o3fs and ofs?

Reply:
`o3fs` has conflicted with some other prefix, so we may need to specify Ozone's prefix by PropertyKey 
 `UNDERFS_OZONE_PREFIXES`

Reply:
Init a configuration instance is very expensive in Alluxio. Do not do that in one method as a temp variable.

In each Alluxio process there's an `AlluxioConfiguration` singleton for you to use. You can just `Configuration.global().getList(PropertyKey.UNDERFS_OZONE_PREFIXES)` when you need it.

Also you should read and parse this only once as a static field in this class.